### ISSUE TITLE
안쓰는 파일은 제거해주세요

### ISSUE THREAD

**Issue by SeolJaeHyeok** [yuna-c/pokemonDex#7]
https://github.com/yuna-c/pokemonDex/blob/b165db66df2bda41fcd4fda77266dc5a94deea7a/src/pages/pokemonApi/API.bak#L1-L114

이 파일은 사용하지 않는 것 같아서 제거해주시면 됩니다!

**Comment by yuna-c** [comment]
네 설재혁튜터님! 이거  api 써서 포켓몬 데이터 가져와서 돌려보고 리팩토링 하고 싶어서 Asset폴더에 넣어 놓았습니다.
